### PR TITLE
[make] full implementation according specification

### PR DESCRIPTION
**Pull request by Wandalen**

### CODE REVIEW

**Comment by Wandalen** [comment]
@jgarzik ready for review

File: m4/tests/integration_test.rs [hunk 1]
```diff
@@ -109,7 +109,7 @@ fn run_command(input: &Path) -> std::process::Output {
 #[test]
 fn test_bsd() {
     init();
-    let output = run_command(Path::new("fixtures/integration_tests/bsd.m4"));
+    let output = run_command(&Path::new("fixtures/integration_tests/bsd.m4"));
```
Comment (jgarzik) [inline m4/tests/integration_test.rs]: merge error.  This PR should not touch m4/

File: plib/src/testing.rs [hunk 2]
```diff
@@ -84,9 +84,6 @@ pub fn run_test(plan: TestPlan) {
     let stdout = String::from_utf8_lossy(&output.stdout);
     assert_eq!(stdout, plan.expected_out);
 
-    let stderr = String::from_utf8_lossy(&output.stderr);
-    assert_eq!(stderr, plan.expected_err);
-
```
Comment (jgarzik) [inline plib/src/testing.rs]: Change rejected -- This change deletes some tests _across all utilities_.

**Review by jgarzik — CHANGES_REQUESTED** [review]
Mostly looks OK.  Some minor changes.

**Comment by Wandalen** [comment]
@jgarzik took into account. is there anything?

**Pull request merged**